portfolio.html

```diff
@@ -259,9 +259,10 @@ <h6 class="connect">LET'S CONNECT</h6>
           <br>
           <input type="email" name="email" id="email" placeholder="Enter your email" required value="natajayanj14@gmail.com">
           <br>
-          <textarea maxlength="500" minlength="2" placeholder="Write your message here" required></textarea>       
+          <textarea maxlength="500" minlength="2" placeholder="Write your message here" required></textarea>        
       </section>
       <button class="more" id="get">Get It Touch</button>
+      </header>
```

Comment:
- [ ] Kindly use the `<form>` element for your form. The form element enables you to add `action`  and `method` attributes to your form. You will use [formspree service](https://formspree.io/html) to generate a unique link that you will add to the `action` attribute as a value so that your form can be submitted successfully
- [ ]  Kindy increase the height of your `<textarea>` and also position your `get in touch` button correctly to match the original design

![image](https://user-images.githubusercontent.com/98453979/205895952-823a32c1-03ee-4446-9583-73aad51a287f.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205896089-fe75058c-f496-47ff-8488-74e94f2fe235.png)